test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectMessageTests.cs

```diff
@@ -25,15 +26,17 @@ public void Constructors(OpenIdConnectMessageTheoryData theoryData)
             TestUtilities.WriteHeader($"{this}.Constructors", theoryData);
             var context = new CompareContext($"{this}.ReadMetadata, {theoryData.TestId}");
             OpenIdConnectMessage messageFromJson;
-            OpenIdConnectMessage messageFromJsonObj;
+            // TODO - these local variable has been left commented out as
```

Comment:
>these local variable has been left commente [](http://example.com/codeflow?start=22&length=43)

do we have a GH issue tracking the work?

Reply:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/2224